### PR TITLE
Feature/info button on legend

### PR DESCRIPTION
**Pull request by weberjavi**
This PR adds an info button on the legend to open the metadata modal.
It also tweaks some of the styles of the legend.
[PIVOTAL](https://www.pivotaltracker.com/story/show/166798171)

### CODE REVIEW

File: src/components/legend/legend-selectors.js [hunk 1]
```diff
@@ -1,16 +1,18 @@
...
 import { legendConfigs as WDPALegendConfigs } from 'constants/protected-areas';
 
+const legendFreeLayers = [FIREFLY_LAYER, BIODIVERSITY_FACETS_LAYER, FEATURES_LABELS_LAYER, GRID_LAYER];
+const isLegendFreeLayer = layerId => legendFreeLayers.some( l => l === layerId);
 
 const getVisibleLayers = createSelector(getActiveLayers, activeLayers => {
   if (!activeLayers.length) return null;
 
-  return activeLayers.filter(layer => layer.id !== FIREFLY_LAYER && layer.id !== BIODIVERSITY_FACETS_LAYER && layer.id !== 'Grid layer');
+  return activeLayers.filter(layer => !isLegendFreeLayer(layer.id));
```
Comment (kfilip94) [inline src/components/legend/legend-selectors.js]: :+1: nice, I was thinking about something like that

**Comment by kfilip94** [comment]
There is no modal-metadata for `Community-based protected areas metadata` and `All protected areas metadata` yet, right?
Screenshot from 2019-06-21 10 34 55

**Comment by kfilip94** [comment]
@aabdaab I fixed a bug with fetching modal metadata for legend items, protected areas. Ready for review :point_up:

**Comment by aabdaab** [comment]
Since we are on the legend topic:
image
Two things here: 
1) if you change the opacity to 0.6, the icon is no longer highlighted to indicate that it's different from 100%.
2) Let's add some padding to the text maybe?

**Comment by kfilip94** [comment]
@aabdaab Both problems solved, ready to review again!
image

**Comment by fannycc** [comment]
Looks good to me, the human pressures ramp it looks weird. In the demo it looks good. Doublecheck maybe?

**Comment by kfilip94** [comment]
@fannycc Sure, I'll create a new pivotal task and I'll do it in an another PR :+1:

**Pull request merged**